robot-name/robot-name_test.spec.js

```diff
@@ -12,9 +12,17 @@ describe("Robot", function() {
   });
 
   it("different robots have different names", function() {
-    var robotOne = new Robot();
-    var robotTwo = new Robot();
-    expect(robotOne.name).not.toEqual(robotTwo.name);
+    var usedNames= {};
+
+    var i = 0,
+        max = 10000;
+
+    for (i; i < max; i++) {
```

Comment:
Since this is one atomic change, I think it would make sense to squash all the commits into a single one. Did I link you to this last time? I don't recall: http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request